lib/ansible/modules/network/f5/bigip_remote_syslog.py

```diff
@@ -0,0 +1,457 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 F5 Networks Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
we now have a one-line GPLv3+ header that we would like modules to use.  Since modules get transferred over the network (not so much for network modules at least for now, but possibly in the future), the one-line version cuts down on bandwidth usage.  http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#copyright

Now that we only support python-2.6+ on the remote side, we're also having people add the standard boilerplate to their modules, right after the license header:

``` python
from __future__ import absolute_import, division, print_function
__metaclass__ = type
```